### PR TITLE
#284:Remove python2 from standard-EXASOL-7.1.0

### PR DESCRIPTION
**Pull request by tomuben**

### CODE REVIEW

File: flavors/standard-EXASOL-7.1.0/flavor_base/build_run/Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ COPY /exaudfclient /exaudfclient
 
 WORKDIR /exaudfclient/extension
 RUN ["/bin/bash", "-c", "source /env && bash build.sh --config no-tty --config optimize --config python --config java --config optimize-r --config fast-binary-both"]
-RUN cp -r -L bazel-bin/* /exaudf && rm -r /exaudf/external/bazel_tools/
+RUN cp -r -L bazel-bin/* /exaudf && rm -r /exaudf/external/bazel_tools/ && rm -r /exaudf/exaudfclient
```
Comment (tkilias) [inline flavors/standard-EXASOL-7.1.0/flavor_base/build_run/Dockerfile:29]: Only, deleting /exaudf/exaudfclient is probably not enough, you also should exclude it from the build by changing `--config fast-binary-both` to  `--config fast-binary-py3`, similar for base_test_build_run/Dockerfile

**Review by tkilias — CHANGES_REQUESTED** [review]
Besides my code comments, you probably need also to cleanup the apt_packages for the flavor_base_deps build step

**Comment by tomuben** [comment]
In order to remove also ```libpython2.7```, we would have to remove ```python3-smbc```.
Dependency chain: python3-smbc => libsmbclient => samba-libs => libpython2.7.

**Review by tkilias — APPROVED** [review]

**Comment by tomuben** [comment]
@tkilias , please check.
Latest changes:
1. Remove the python-smbc chec
2. Revert of using the EXASOL Websocket pinned version.

**Pull request merged**